docs/src/main/sphinx/release/release-473.md

```diff
@@ -1,5 +1,9 @@
 # Release 473 (19 Mar 2025)
 
+```{warning}
+This release is broken and should not be used.
```

Comment:
Maybe include something about GROUP BY and DISTINCT producing incorrect results

Reply:
Updated phrasing and included an issue link, that look alright?